app/experimenter/experiments/tests/test_models/test_models_nimbus.py

```diff
@@ -372,6 +372,30 @@ def test_targeting_with_locales_and_countries(self):
             ),
         )
 
+    def test_targeting_with_locales_and_countries_mobile(self):
+        locale_ca = LocaleFactory.create(code="en-CA")
+        locale_ro = LocaleFactory.create(code="ro")
+        locale_de = LocaleFactory.create(code="de")
+        locale_en = LocaleFactory.create(code="en")
+        targeting_config = NimbusExperiment.TargetingConfig
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE_APPROVE,
+            application=NimbusExperiment.Application.FENIX,
+            firefox_min_version=NimbusExperiment.Version.NO_VERSION,
+            firefox_max_version=NimbusExperiment.Version.NO_VERSION,
+            targeting_config_slug=targeting_config.TARGETING_MOBILE_NEW_USER,
+            channel=NimbusExperiment.Channel.NO_CHANNEL,
+            locales=[locale_ca, locale_ro, locale_de, locale_en],
+        )
+        self.assertEqual(
+            experiment.targeting,
+            (
+                "is_already_enrolled || days_since_install < 7 "
+                "&& (locale in ['en-CA'] || 'de' in locale || 'en' in locale "
+                "|| 'ro' in locale)"
```

Comment:
@jhugman @travis79 Can you please help verify that this looks correct as a workaround to match Experimenter's locale codes to the mobile ones?

Reply:
Supplemental @jrbenny35 is there a way we can test/verify this against a mobile client before we deploy it?

Reply:
Yes, this format looks correct to me for `en-CA` as to what we would currently have in the `locale` of our  context.

Reply:
Looking at the checks here, https://github.com/mozilla/application-services/pull/4857 this looks fine